docs/repos.md

```diff
@@ -4,77 +4,116 @@
+
+<div class='projects' markdown='1'>
+
+- ### [GraphKB Data Loaders](https://github.com/bcgsc/pori_graphkb_loader)
+
+    ![graphkb loaders](./images/graph-icon_outline.svg)
+
+    GraphKB loaders is responsible for all data import into GraphKB. Automatic Import modules are provided for a variety of external ontologies and knowledgebases such as: Ensembl, Entrez Genes, RefSeq, HGNC, Disease Ontology, NCI Thesaurus, CIViC, DoCM, etc.
+
+</div>
+
+## Python Adaptors
+
+The popularity of python in bioinformatics makes it one of the top choices for adaptors. These adaptors are written to help users integrate PORI into their existing bioinformatic workflows. They are published and installed via pip.
+
+```bash
+pip install graphkb ipr
+```
+
+A developer reference for both packages including the function signatures and package details can be found in the developer reference section [here](../developer_reference).
+
+<div class='projects' markdown='1'>
+
+- ### [GraphKB Python Adaptor](https://github.com/bcgsc/pori_graphkb_python)
 
     ![graphkb python](./images/graph-icon_outline.svg)
 
     Python adapter package for querying the GraphKB API. See the related
     [user manual](../graphkb/python/docs) for instructions on incorporating
     this into custom scripts.
 
-- ## [IPR API](https://github.com/bcgsc/pori_ipr_api)
-
-    ![ipr api](https://www.bcgsc.ca/gsc-logos/icon/logo-circle.png)
-
-    The Integrated Pipeline Reports (IPR) REST API manages data access to the IPR database. The API is
-    responsible for storing and server all data for reports.
+- ### [IPR Python Adaptor](https://github.com/bcgsc/pori_ipr_python)
 
-- ## [IPR client](https://github.com/bcgsc/pori_ipr_client)
+    ![ipr python](./images/wrench.svg)
 
-    ![ipr client](./images/pori-ipr-main-report-page.png)
+    Python adapter for generating reports uploaded to the IPR API. This python tool takes in variant inputs as tab-delimited files and annotates them using GraphKB. The resulting output is uploaded to IPR as a report. Additional report content such as images and metadata can be passed to be included in the report upload.
 
-    The IPR client is front-end web application which consumes data from the IPR API.
-    The primary function is the production and management of genomic reports.
+- ### [PORI cBioportal](https://github.com/bcgsc/pori_cbioportal)
 
-- ## [IPR Python Adaptor](https://github.com/bcgsc/pori_ipr_python)
+    ![pori cbioportal](https://frontend.cbioportal.org/reactapp/images/369b022222badf37b2b0c284f4ae2284.png)
 
-    ![ipr python](./images/wrench.svg)
+    This python adaptor is intended to demonstrate creating a PORI report using data exported from a cBioportal instance. It uses the expression, copy number, fusion, and small mutation data as well as available metadata to complete the reports.
```

Comment:
'adaptor' and 'adapter' are both used throughout these docs; should probably stick with one

Reply:
'adaptor', probably

Reply:
(see other comment)